src/ucp/wireup/select.c

```diff
@@ -953,6 +1013,15 @@ ucs_status_t ucp_wireup_select_lanes(ucp_ep_h ep, const ucp_ep_params_t *params,
                                                           address_list,
                                                           lane_descs,
                                                           key->num_lanes);
+    /* Cut slowest rndv lanes by configuration value */
+    for (lane = ep->worker->context->config.ext.rndv_rails; lane < UCP_MAX_RAILS; lane++) {
+        key->rndv_lanes[lane] = UCP_NULL_LANE;
```

Comment:
indentation

Reply:
what is wrong with indentation here?

Reply:
need one space before 'key->rndv_lanes[lane] = UCP_NULL_LANE;' to align to the for loop.